as/http.js

```diff
@@ -182,6 +183,16 @@ TChannelHTTP.prototype.sendRequest = function send(treq, hreq, options, callback
 
     treq.headers.as = 'http';
     if (treq.streamed) {
+        self.channel.emitFastStat(self.channel.buildStat(
+            'tchannel.outbound.http-handler.request-build-latency',
+            'timing',
+            self.channel.timers.now() - start,
+            new stat.HTTPHanlderBuildLatencyTags(
+                treq.serviceName,
+                treq.callerName,
+                treq.streamed
+            )
+        ));
         return treq.sendStreams(arg1, arg2, hreq, onStreamResponse);
```

Comment:
you want to call `sendStreams()` and then emit stat because you want the time it takes to call sendStreams() to be included in latency